pkgs/servers/dns/https-dns-proxy/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, c-ares
+, curl
+, libev
+}:
+
+stdenv.mkDerivation rec {
+  pname = "https_dns_proxy";
+  # there are no stable releases (yet?)
+  version = "git-20200301";
```

We commonly use this scheme:

```suggestion
  version = "unstable-2020-03-01";
```